### ISSUE TITLE
X Deaktivieren

### ISSUE THREAD

**Issue by Mike-Dobl** [modelleicher/FS19_simpleIC#54]
Kurze Frage, wie kann man die weißen X deaktivieren.
neue Tastenbelegung brachte nichts.

LG
mike

**Comment by modelleicher** [comment]
Mit der IC: an/aus schalten Taste kann man IC an bzw. aus schalten. Je nach Mod-Kombination muss man wohl manchmal das Fahrzeug ausgewählt haben. Tastendoppelbelegung ist natürlich auch eine Option

**Comment by Mike-Dobl** [comment]
Danke für die schnelle Rückmeldung.
Jedoch, dachte ich mir dass auch schon und habe es merfach probiert...
mein Fall:
Traktor/Unimog mit Simple IC (ohne geräte), lassen mich auch nicht die X deaktivieren.
Wenn nur die Zugmaschine vorhanden ist kann man ja nichts mehr auswählen....

Darum meine Frage...

**Comment by Suschi29** [comment]
Moin 
Ich hatte das gleiche Problem.
Nachdem ich denn Mod FS19_TowBarWeight raus genommen habe hat wieder alles Funktioniert.

**Comment by modelleicher** [comment]
Ja, das Script das für die Abschleppstangen & Gewichte genutzt wird macht da Probleme..
